### ISSUE TITLE
sphinx-apidoc documentation has a bad comma after EXCLUDE_PATTERN

### ISSUE THREAD

**Issue by Jacob-Stevens-Haas** [sphinx-doc/sphinx#6674]
TL; DR: remove the comma after \`_EXCLUDE\_PATTERN_\` in the sphinx-apidoc man page and help text.

**Describe the bug**
__sphinx-apidoc__ sees a comma at the end of exclude pattern as part of fnmatch style string.  Recommend removing the comma from the syntax on the man page and --help response, or changing the program to ignore a trailing comma in exclude patterns

**To Reproduce**
```
> mkdir mwe
> cd mwe
> mkdir docs
> type nul > ignore1.py # aka touch ignore1.py
> type nul > ignore2.py
> type nul > keep.py
> cd docs
> sphinx-quickstart #Choose separate build & source directories
> sphinx-apidoc -o source ../ ../ignore1.py, ../ignore2.py
Creating file source\ignore1.rst.
Creating file source\keep.rst.
Creating file source\modules.rst.
```

**Expected behavior**
Expect either the documentation to lose the comma, or sphinx-apidoc to treat the trailing comma in `../ignore1.py,` as a separator between arguments and not part of the match string.  Currently, the correct command is
```
> sphinx-apidoc -f -o source ../ ../ignore1.py ../ignore2.py
Creating file source\keep.rst.
Creating file source\modules.rst.
```

**Environment info**
- OS: Windows 10
- Python version: 3.7.3
- Sphinx version: 2.1.2
- Sphinx extensions:  none.  Bare bones from `sphinx-quickstart`
- Extra tools: none

**Additional context**
I hope this doesn't come across as cavil, but it caused a lot of confusion for my colleague and me. The [man page for sphinx-apidoc][spa] writes the syntax of the command as:

__sphinx-apidoc__ [_OPTIONS_] -o <_OUTPUT\_PATH_> <_MODULE\_PATH_> [_EXCLUDE\_PATTERN_, …]

That comma at the end of \`_EXCLUDE\_PATTERN_,\` confused me.  It isn't necessary; sphinx-apidoc sees the comma as part of the fnmatch style string, meaning that only my last exclude pattern was matching the files I wanted it to.  For comparison, the [man page for sphinx-build][spb] does not use the spurious comma:

__sphinx-build__ [_options_] <_sourcedir_> <_outputdir_> [_filenames_ …]

There's a [stackoverflow question][so] about proper syntax of documentation. The approved answer cites [Microsoft's command line syntax key][msft], which does not include the comma, and [IBM style][ibm], which does, but the command presumably treats the commas as separators, not as part of the argument.  Moreover, an additional answer cites the [POSIX standard][posix], which does not include commas.  Another answer mentions that GNU coding standards are vague re: [--help][help] and [man][man] pages

[spa]: https://www.sphinx-doc.org/en/master/man/sphinx-apidoc.html
[spb]: https://www.sphinx-doc.org/en/master/man/sphinx-build.html
[so]: https://stackoverflow.com/questions/9725675/is-there-a-standard-format-for-command-line-shell-help-text
[msft]: https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/command-line-syntax-key
[ibm]: https://www.ibm.com/support/knowledgecenter/en/STAV45/com.ibm.sonas.doc/sonas_manpages_syntax.html
[posix]: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html#tag_12_01
[help]: https://www.gnu.org/prep/standards/standards.html#g_t_002d_002dhelp
[man]: https://www.gnu.org/prep/standards/standards.html#Man-Pages
